setup.py

```diff
@@ -40,14 +40,15 @@
 from setuptools import Extension, setup, Command
 from setuptools.command.build_ext import build_ext as _build_ext
 
+PY3 = sys.version_info[0] == 3
```

Comment:
This line is repeated in two different files. Would it makes sense to set it once somewhere common and just import it where needed? Maybe `pycbc.PY3`?